src/solution_part2.kt

```diff
@@ -1,3 +1,9 @@
-fun main(){
- print("I love Kotlin!!!!")
-}
\ No newline at end of file
+fun main() {
+ val healthPointP1 = 94
+ val healthPointP2 = 45
+ if (healthPointP1 >= 90) {
+  println("you are fine")
+  if (healthPointP2 <= 50)
+   println("you need to see a Doctor!!!")

```

Comment:
wrong logic 
 fun main(){
  val PlayerHealthPoints =90
 when(PlayerHealthPoints){
    in 50..89 -> println("the player's health is fine")
    in 0..49 -> println("the player's health is awful")
   else -> println("Score not valid")
  }}